### PR TITLE
Create coding_boost.md

### PR DESCRIPTION
**Pull request by amitmerchant1990**
Added my 2 paise of chhoti lekin moti baatein. 😆

### CODE REVIEW

File: _posts/2017-02-16-coding-boost.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+layout: post
+title:  "Coding boost"
+date:   2017-01-16 10:46:00
```
Comment (chinchang) [inline _posts/2017-02-16-coding-boost.md]: should be february in the date

Comment (amitmerchant1990) [reply _posts/2017-02-16-coding-boost.md]: Already fixed that. 😸

**Comment by chinchang** [comment]
@amitmerchant1990 Thanks for your choti-moti baat :) Its live!

**Comment by amitmerchant1990** [comment]
yay! Thanks @chinchang

**Pull request merged**